Doc/library/sys.rst

```diff
@@ -744,9 +744,11 @@ always available.
    version of the Python *language* to which the currently running
    interpreter conforms, which ``sys.version_info`` represents.  For
    example, for PyPy 1.8 ``sys.implementation.version`` might be
-   ``sys.version_info(1, 8, 0, 'final', 0)``, whereas ``sys.version_info``
-   would be ``sys.version_info(2, 7, 2, 'final', 0)``.  For CPython they
-   are the same value, since it is the reference implementation.
+   ``sys.version_info(major=1, minor=8, micro=0, releaselevel='final', serial=0)``,
```

Comment:
The current writing LGTM. `sys.version_info(1, 8, 0, 'final', 0)` is not a repr, this is a constructor. `sys.version_info(major=1, minor=8, micro=0, releaselevel='final', serial=0)` looks too verbose.

Reply:
Sorry, I thought it should look the way it is in the console when `sys.version_info` is typed.  I didn't realize it was a constructor.  I'll withdraw the PR.